da-contracts/contracts/ValidiumL1DAValidator.sol

```diff
@@ -24,6 +24,9 @@ contract ValidiumL1DAValidator is IL1DAValidator {
     }
 
     function supportsInterface(bytes4 interfaceId) external pure returns (bool) {
+        if (interfaceId == this.supportsInterface.selector) {
```

Comment:
BTW, we with @koloz193 decided to remove the support for EIP165 here... But since it is present here, okay let's fix it an remove later

Reply:
Where did this happen btw? ( so that  I don't miss it in the future)

Reply:
Hmm, I can not find it, probably it was decided in dms only. Better to double check with him when he comes back